### PR TITLE
Transform header and footer data functions into data objects

### LINKED ISSUE #559: Transform header and footer data functions into data objects
We currently have every text of the website inside the `data` folder, separated in different files.

The one for the header is the `data/header.js`. It exports a function called `getHeaderData` which doesn't receive any argument. This is legacy code, and we would like to rework it in order to have an object instead. 

What we have to do is: 
- Transform the `getHeaderData`  function of the `data/header.js` file into an object
- Rename `getHeaderData` into `headerData` in every file that uses it (including the `stories` folder used by Storybook)

Same thing for the Footer (`getFooterData` becomes `footerData` + transform function into object)

### PR DESCRIPTION
**Pull request by simonpankovski**
# Pull Request

## Related issue
Fixes #559 

## What does this PR do?
- Transforms the getFooterData and getHeaderData functions of the data/header.js and data/foother.js files into objects.
- Renames the lines where the functions were mentioned into their object representations (footerData and headerData, respectively).

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/blocks/Footer/index.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const LogoSection = styled.div`
 `
 
 const Footer = props => {
-  const footerProps = getFooterData()
+  const footerProps = footerData
```
Comment (mdubus) [inline src/blocks/Footer/index.js:37]: Can you rework this to remove the `footerProps` useless variable ? :)

File: src/blocks/Header/index.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const HeaderWrapper = styled.div`
 `
 
 const Header = props => {
-  const headerProps = getHeaderData()
+  const headerProps = headerData
```
Comment (mdubus) [inline src/blocks/Header/index.js:33]: Same as above 🙏

File: src/stories/Menu.stories.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ export default {
 
 export const Default = () => {
   const menuState = useMenuState()
-  const headerProps = getHeaderData()
+  const headerProps = headerData
```
Comment (mdubus) [inline src/stories/Menu.stories.js:17]: Same

File: src/stories/Menu.stories.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ export const Default = () => {
 
 export const Animated = () => {
   const menuState = useMenuState({ animated: 300, loop: true })
-  const headerProps = getHeaderData()
+  const headerProps = headerData
```
Comment (mdubus) [inline src/stories/Menu.stories.js:36]: Same

**Review by mdubus — CHANGES_REQUESTED** [review]

File: src/blocks/Footer/index.js (hunk 1 above)
Comment (simonpankovski) [reply src/blocks/Footer/index.js:37]: Yes, I'll get on it now

**Review by mdubus — APPROVED** [review]
That's perfect, thanks a lot for your work! 🥳  🦕

bors merge

**Pull request merged**